src/OnboardingSPA/pages/Steps/GetStarted/SiteTypeSetup/SecondarySite/index.js

```diff
@@ -101,71 +102,75 @@ const StepPrimarySetup = () => {
 						) }
 					/>
 				</div>
-
-				<div className="nfd-setup-secondary-categories">
-					<div className="nfd-card-category-wrapper">
-						<div className="category-scrolling-wrapper">
-							<span
-								className="icon"
-								style={ {
-									backgroundImage: categoriesArray[ 0 ].icon,
-								} }
-							/>
-							<p className="categName">
-								{ ' ' }
-								{ categoriesArray[ 0 ].name }
-							</p>
+				<Animate type="fade-in" after={ categoriesArray[ 0 ]?.subCategories && selectedCategoryInStore }>
+					<div className="nfd-setup-secondary-categories">
```

Comment:
`selectedCategoryInStore` need to have a conditional check, the page does not load when the user has not chosen any option.
Similarly in primary site step

Reply:
have removed "selectedCategoryInStore" from check cause there is a state where that value can be empty and when the value is empty the animate component hides the content

Reply:
Added it back with a null check should work now